app/code/community/Nexcessnet/Turpentine/Model/Varnish/Configurator/Abstract.php

```diff
@@ -746,13 +746,18 @@ protected function _vcl_get_probe($probeUrl) {
             .probe = {
                 .request =
                     "GET {{probe_path}} HTTP/1.1"
-                    "Host: {{probe_host}}"
+                    "Host: {{probe_host}}"{{probe_https}}
```

Comment:
So you mean having Host: {{probe_host}} with a value for {{probe_host}} will be the same as having Host: {{probe_host}} with an empty string? I'd like to have a cleaner approach to take both http an https into account.